nestkernel/event.h

```diff
@@ -827,13 +843,20 @@ class GapJEvent : public SecondaryEvent
   static void
   set_syn_id( const synindex synid )
```

Comment:
Yes, `set_...` is only called when the connection type is registered. But I guess it will also be called for the labeled version of the synapse, at least if it is implemented in the same way as for primary connection models. So maybe we could add an `assert( not supported_syn_id( synid ) )` to the `add_...`-function and leave `set_...` without assert. Or we could add the same `assert` for both, what do you think?